azure/services/scalesets/scalesets.go

```diff
@@ -59,6 +58,9 @@ type (
 	}
 )
 
+// ScalesetsServiceName is the name of the scalesets service.
+const ScalesetsServiceName = "scalesets"
```

Comment:
is this used outside the scaleset package? If not no need to make it public. Also since scalesets is the name of the package already should be avoid the stutter

```suggestion
const servicename = "scalesets"
```